vm/parser.ts

```diff
@@ -804,6 +804,11 @@ module J2ME {
     }
 
     get isNative(): boolean {
+      if (!release) {
+        if (Native[this.implKey]) {
+          return true;
+        }
+      }
```

Comment:
Added a description in https://github.com/marco-c/j2me.js/commit/c6f9ebfaf872f9368ca23e994fc41dfbc5669b14.